.github/workflows/release.yml

```diff
@@ -18,21 +18,36 @@ jobs:
         id: get_version
         uses: battila7/get-version-action@90eb8fc70f6dfcf3f9b95ed8f164d2c05038e729 # v2.2.1
 
+      #Substitute the Manifest and Download URLs in the module.json
```

Comment:
suggestion: use `jq` instead of external action

I do not like us adding additional dependencties when this could be done by shell job (maybe during zip step) that uses jq to replace those variables. One less external workflow to worry about

Reply:
Agreed, plus this action uses some GHA syntax that will be deprecated at some point. For now I just reinstated this step which we used to have from before 1.15.0. I'll make a follow up PR to change it after this set of PRs are merged, so as to avoid making the git tree more confusing.